cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -334,7 +334,7 @@ def __init__(self, *args, **kwargs):
         self._clusters = None
         self._dc_cache = {}
         self._ds_regex = None
-        self.additional_endpoints.append([
+        self.additional_endpoints.extend([
```

Comment:
This is here by accident, rigth?

Reply:
The original queens patch that Marius created had extend, not append, so I just fixed that too.

Reply:
In the same commit? I don't like that.

Reply:
removed